FirebaseInstallations/Source/Library/FIRInstallationsItem.h

```diff
@@ -44,8 +44,8 @@ NS_ASSUME_NONNULL_BEGIN
 @property(nonatomic, nullable) FIRInstallationsStoredAuthToken *authToken;
 @property(nonatomic, assign) FIRInstallationsStatus registrationStatus;
 
-/// Instance ID checkin data imported from IID checkin store as a part of IID migration.
-@property(nonatomic, nullable) FIRInstallationsStoredIIDCheckin *IIDCheckin;
+/// Instance ID default auth token imported from IID store as a part of IID migration.
+@property(nonatomic, nullable) NSString *IIDAuthToken;
```

Comment:
It's more of a IIDToken, but I'm okay to leave it if it's a hassle to rename.

Reply:
I renamed it to `IIDDefaultToken`